src/workloads/execution/ProjectSort.yml

```diff
@@ -0,0 +1,152 @@
+  performance both without and without an index.
+  Important Metrics: timers.dur, timers.total on the aggregate commands. The index build is to test the performance
+  of the aggregation with/without an index.
+Keywords:
+  indexes
+  collections
+  aggregation
+  sort
+
+GlobalDefaults:
+  aggregate: &collName Collection0
+  pipeline: &aggPipe [{$project: {array: 0}}, {$sort: {integer: 1}}]
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: &docCount 100000
+    BatchSize: *docCount
+    Document:
+      integer: &integer {^RandomInt: {min: 1, max: 10000}}
+      double: &double {^RandomInt: {distribution: geometric, p: 0.1}}
+      string: &string {^RandomString: {length: 16}}
+      array:
+      - *integer
+      - *integer
+      - subInteger: *integer
+        subString: *string
+        subArray:
+        - *integer
+        - *integer
+      - subInteger: *integer
+        subString: *string
+        subArray:
+        - *integer
+        - *integer
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: BuildIndex
+  Type: RunCommand
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *collName
+        indexes: &IndexSpec [{key: {"integer": 1}, name: "int_1"}]
+  - *Nop
+  - *Nop
+
+
+- Name: Quiesce
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+
+- Name: AggregateNoIndex
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: &aggRepeat 10
+    Database: *db
+    Operations:
+    - OperationMetricsName: AggregateNoIndex
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *collName
+        pipeline: *aggPipe
+        cursor: {batchSize: *docCount}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: AggregateIndex
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: *aggRepeat
+    Database: *db
+    Operations:
+    - OperationMetricsName: AggregateIndex
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *collName
+        pipeline: *aggPipe
+        cursor: {batchSize: *docCount}
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - replica
+      - replica-all-feature-flags
+      - shard-lite
+      - standalone
```

Comment:
Question: do we really need all these mongodb setups? Would standalone and standalone-classic-query-engine suffice?

Reply:
Done.

Reply:
:+1: